client/src/hooks/useTimeInMutation.ts

```diff
@@ -27,9 +29,14 @@ const useTimeInMutation = (): returnType => {
         })
       },
       onSuccess: async () => {
-        await queryClient.invalidateQueries({
-          queryKey: ['GET_USER_QUERY']
-        })
+        await Promise.all([
+          queryClient.invalidateQueries({ queryKey: ['GET_USER_QUERY'] }),
```

Comment:
Try this if it is possible?

```suggestion
          queryClient.invalidateQueries({ queryKey: ['GET_USER_QUERY', 'GET_EMPLOYEE_TIMESHEET'] }),
```